WMF Framework/Localization.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+public func WMFLocalizedString(_ key: String, siteURL: URL? = nil, bundle: Bundle = Bundle.wmf_localization, value: String, comment: String) -> String {
```

Comment:
New localization function for swift - by matching the method signature of `NSLocalizedString` we can use the built in strings file generation tools.